server/core/terraform/terraform_client_internal_test.go

```diff
@@ -344,7 +344,7 @@ func TestDefaultClient_RunCommandAsync_ExitOne(t *testing.T) {
 	_, outCh := client.RunCommandAsync(ctx, tmp, []string{"dying", "&&", "exit", "1"}, map[string]string{}, nil, "workspace")
 
 	out, err := waitCh(outCh)
-	ErrEquals(t, fmt.Sprintf(`running 'sh -c "echo dying && exit 1"' in '%s': exit status 1`, tmp), err)
+	ErrEquals(t, fmt.Sprintf(`running 'sh -c echo dying && exit 1' in '%s': exit status 1`, tmp), err)
```

Comment:
Removing the quotes here makes this harder to read. `sh -c echo dying && exit 1` means a different thing than `sh -c "echo dying && exit 1"`

Reply:
In the JSON logs, the `msg` property currently looks like this:

```json
"msg":"successfully ran 'sh -c \"/root/.atlantis/bin/terraform1.9.8 init -input=false -upgrade\"' in '/root/.atlantis/repos/sheather/test/31/default/live/aws/stack11'"
```

which with the escaped double quotes is very confusing to read, so I removed them to give this:

```json
"msg":"successfully ran 'sh -c /root/.atlantis/bin/terraform1.9.8 init -input=false -upgrade' in '/root/.atlantis/repos/sheather/test/31/default/live/aws/stack11'"
```

I can replace it with single quotes, or backticks instead:

**Single Quotes**

```json
"msg":"successfully ran 'sh -c '/root/.atlantis/bin/terraform1.9.8 init -input=false -upgrade'' in '/root/.atlantis/repos/sheather/test/31/default/live/aws/stack11'"
```

**backticks**

```json
"msg":"successfully ran 'sh -c `/root/.atlantis/bin/terraform1.9.8 init -input=false -upgrade`' in '/root/.atlantis/repos/sheather/test/31/default/live/aws/stack11'"
```

Thoughts?

Reply:
Ah yeah, that's tough.

My personal view is that if you're logging to JSON you can expect `\"`s, and that it's still useful to be able to recover what command was actually ran.

But I could definitely see it going either way, and am fine with any of the above solutions

Reply:
I've added single quotes.

Reply:
Works for me!